package.json

```diff
@@ -36,6 +36,7 @@
     "ember-export-application-global": "^2.0.0",
     "ember-load-initializers": "^1.0.0",
     "ember-resolver": "^4.0.0",
+    "ember-router-service-polyfill": "^1.0.2",
```

Comment:
I believe this needs to be a dep not a devDep

Reply:
done, thanks for the review